libft/tests/map/mapss/ft_mapss_print_test.c

```diff
@@ -6,11 +6,87 @@
 /*   By: inikulin <inikulin@student.42berlin.de>    +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/08/13 21:34:36 by inikulin          #+#    #+#             */
-/*   Updated: 2024/08/13 21:49:35 by inikulin         ###   ########.fr       */
+/*   Updated: 2024/10/27 00:39:00 by inikulin         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
-void	ft_mapss_print_test(void)
+#include "../../tests_internal.h"
+
+#define START 0
+#define SZ 1
+//#define DEBUG
+
+static void	catch(char* fname, int *out, int *save)
+{
+	remove(fname);
+	fflush(stdout);
+	*out = open(fname, O_RDWR|O_CREAT, 0600);
+	assert (-1 != *out);
+	*save = dup(fileno(stdout));
+	assert (-1 != dup2(*out, fileno(stdout)));
+}
+
+static void	finally(int *out, int *save)
 {
-	return ;
+	fflush(stdout);
+    close(*out);
+	dup2(*save, fileno(stdout));
+	close(*out);
+}
+
+typedef struct s_testcase
+{
+	char	***content;
+	int		sz;
```

Comment:
norminette

Reply:
what do you mean, norminette? it looks good with the correct font, also this is a test file, it's not checked by norminette
![image](https://github.com/user-attachments/assets/e4b1e575-9097-449f-9420-ffb433506c2d)

Reply:
by norminette here i mean that int sz right alligned in github
![Screenshot from 2024-11-08 10-43-29](https://github.com/user-attachments/assets/626f907c-d511-4728-91ba-43535ed9ef81)

Reply:
GitHub uses a wrong font, so indents here are often irrelevant to what norminette actually expects